pom.xml

```diff
@@ -29,7 +29,7 @@
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>display-url-api</artifactId>
-      <version>0.2</version>
+      <version>0.5</version>
```

Comment:
Bumping the minimum here as there have been many improvements and bug fixes which are beneficial to this working correctly since 0.1